pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -411,6 +415,29 @@ func hasMemoryAndCPUInstUsage(info *cadvisorapiv2.ContainerInfo) bool {
 	return cstat.CpuInst.Usage.Total != 0 && cstat.Memory.RSS != 0
 }
 
+// isContainerTerminated returns true if the specified container info has
+// both zero CPU instantaneous usage zero/memory RSS usage/zero network usage,
+// and false otherwise.
+func isContainerTerminated(info *cadvisorapiv2.ContainerInfo) bool {
+	if !info.Spec.HasCpu && !info.Spec.HasMemory && !info.Spec.HasNetwork {
+		return true
+	}
+	cstat, found := latestContainerStats(info)
+	if !found {
+		return true
+	}
+	if (cstat.CpuInst == nil || cstat.Memory == nil) && cstat.Network == nil {
```

Comment:
the comment above the function doesn't seem to match his condition. Shouldn't they all be &&

Reply:
Hi, @haircommander thanks for review, I have updated the comment.